docs/src/get-started/concepts.rst

```diff
@@ -0,0 +1,239 @@
+.. _userdoc-core-concepts:
+
+Core concepts
+=============
+
+This page explains the main types and core concepts you need to understand to
+use equistore, without tying it to how a specific library is using equistore to
+store it's own data.
```

Comment:
```suggestion
use equistore. We discuss equistore without the context
of any specific library, but provide as example, but provide 
an illustrative example later with SOAP to give further context.
```

While staying general at first is great, people may really benefit from one concrete demonstration.

Reply:
Agree with that, but I would put the example in a separate page (with a link to it from this one)

Reply:
Agreed!